packages/vite-plugin-cloudflare/src/workerd-custom-import.cts

```diff
@@ -0,0 +1,10 @@
+module.exports = async (path: string) => {
+  const result = await import(path);
+  // CJS modules don't provide named exports properly in workerd, the named values can be
+  // found under `default` instead, so here we spread default in the returned object
+  // (this should get fixed soon: https://github.com/cloudflare/workerd/pull/2194)
```

Comment:
Is this actually a workerd issue though? I thought named exports from CJS modules are always under `default` when imported in ESM.

Reply:
I am quite sure it is a workerd issue

named exports imported from ESM can be found under `default`, but can also be found as named exports, that works in node.js but not in workerd that's the issue here

For example, the following code does work in node.js but would not work in workerd:
![Screenshot 2024-10-09 at 14 35 51](https://github.com/user-attachments/assets/576335e3-d1af-43f8-b17e-0b1c9a27a35e)

Reply:
oh wait... https://github.com/cloudflare/workerd/pull/2194 got merged... so I should double check, maybe this is actually no longer the case!

Reply:
yeah It's no longer needed! 🚀 

Thanks for bringing this up, this allowed me to greatly simplify the code! 💪